test/prompt.test.ts

```diff
@@ -0,0 +1,33 @@
+import cli from '../src'
+
+import {expect, fancy} from './fancy'
+
+describe('prompt', () => {
+  after(function () {
+    (global as any).asyncDump()
+  })
+  fancy
+  .stdout()
+  .stderr()
+  .end('requires input', async () => {
+    const promptPromise = cli.prompt('Require input?')
+    process.stdin.emit('data', '')
+    process.stdin.emit('data', 'answer')
+    const answer = await promptPromise
+    await cli.done()
+    expect(answer).to.equal('answer')
+  })
+
+  fancy
+  .stdout()
+  .stderr()
+  .end('does not require input', async () => {
```

Comment:
The above test works great, however, the following test hangs.... still not sure why.